src/dataHandler.py

```diff
@@ -63,6 +64,13 @@ def send_strain_gauges(self, sendingTo):
         return
     
     def send_all_data(self, sendingTo):
```

Comment:
```suggestion
    def send_all_data(self, sendingTo, destination):
```
Destination can = name of file so we dont need to hardcode it in